client/packages/common/src/utils/formatters/formatters.ts

```diff
@@ -31,4 +32,6 @@ export const Formatter = {
     const date = dateString ? new Date(dateString) : null;
     return date && isValid(date) ? format(date, "yyyy-MM-dd' 'HH:mm:ss") : '';
   },
+  logTypeTranslation: (logType: string): LocaleKey =>
```

Comment:
ah. yes - even I think this is a little OTT, but have done it anyway. It just looks nicer in the JSON file 🤷